hazelcast/src/main/resources/hazelcast-config-3.8.xsd

```diff
@@ -65,7 +65,7 @@
                 <xs:element name="lite-member" type="lite-member" minOccurs="0" maxOccurs="1"/>
                 <xs:element name="hot-restart-persistence" type="hot-restart-persistence" minOccurs="0" maxOccurs="1"/>
                 <xs:element name="distributed-classloading" type="distributed-classloading" minOccurs="0" maxOccurs="1"/>
-                <xs:element name="cardinality-estimator" type="cardinality-estimator-service" minOccurs="0" maxOccurs="unbounded"/>
+                <xs:element name="cardinality-estimator" type="cardinality-estimator" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
Yes, chose to use without -service, to keep it consistent with other data structures like map etc.